data-serving/scripts/converters.py

```diff
@@ -343,14 +348,89 @@ def convert_outbreak_specifics(id: str, reported_market_exposure: str,
         normalized = convert_bool(lives_in_wuhan)
         if normalized is not None:
             outbreak_specifics['livesInWuhan'] = normalized
-    except (ValueError):
+    except ValueError:
         logging.warning(
             '[%s] [outbreakSpecifics[livesInWuhan]] invalid value %s', id,
             lives_in_wuhan)
 
     return outbreak_specifics if outbreak_specifics else None
 
 
+def convert_location_string(location_str: str) -> Dict[str, Any]:
+    if pd.isna(location_str) or location_str.lower() == 'no' or location_str.lower() == 'none':
+        return None
+    if type(location_str) is not str:
+        raise ValueError('location is not a string')
```

Comment:
Error handling is getting a little hard to track in the file. In top-level converters we're catching errors and logging. But, there isn't a totally clear scheme delineating top-level vs. private/sub-converters. Can you think of a clearer way to handle errors?

Reply:
agreed -- please get excited for a forthcoming refactor into converters, parsers, and utils, with clearly defined roles